com.sayonara_player.Sayonara.json

```diff
@@ -0,0 +1,188 @@
+{
+    "app-id": "com.sayonara_player.Sayonara",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "sayonara",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.mpris.MediaPlayer2.Player",
+        "--own-name=org.mpris.MediaPlayer2.sayonara",
+        "--own-name=org.kde.*",
+        "--device=all",
+        "--filesystem=xdg-run/gvfs"
```

Comment:
Is this actually needed?

Reply:
Yes, the app uses MPRIS2

Reply:
I mean `--filesystem=xdg-run/gvfs` here specifically, MPRIS2 is fine. (No idea why GitHub shows so many lines.)

Reply:
The intention here was to allow access to mounted drives (e.g. smb) using gvfs - removed this for now because it didn't work anyway, working on it later.

Reply:
For gvfs you also need a dbus permission.